study-builder/studies/pancan/study-workflows.conf

```diff
@@ -28,25 +28,25 @@
           "type": "STUDY_REDIRECT",
           "studyGuid": "cmi-mbc",
           "redirectUrl": ${mbcRedirectUrl},
-          "expression": ${_pex.has_breast}"&&"${_pex.is_advanced_breast}"&&"${_pex.is_lang_en},
+          "expression": ${_pex.is_self_only}"&&"${_pex.has_breast}"&&"${_pex.is_advanced_breast}"&&"${_pex.is_lang_en},
```

Comment:
Need to update these?

Reply:
flow chart has a 3rd diff flow if self + child are chosen and that doesn't have any mention of study-redirects.
we can update once we get clarified.

Reply:
will update here